src/System Application/App/File System/src/Account/FileAccounts.Page.al

```diff
@@ -0,0 +1,312 @@
+                    end;
+                }
+                field(NameFieldLookup; Rec.Name)
+                {
+                    ToolTip = 'Specifies the name of the account.';
+                    Visible = IsInLookupMode;
+                }
+                field(DefaultField; DefaultTxt)
+                {
+                    Caption = 'Default';
+                    ToolTip = 'Specifies whether the file account will be used for all scenarios for which an account is not specified. You must have a default file account, even if you have only one account.';
+                    Visible = not IsInLookupMode;
+                }
+                field(FileConnector; Rec.Connector)
+                {
+                    ToolTip = 'Specifies the type of file extension that the account is added to.';
+                    Visible = false;
+                }
+            }
+        }
+
+        area(FactBoxes)
+        {
+            part(Scenarios; "File Scenarios FactBox")
+            {
+                Caption = 'File Scenarios';
+                SubPageLink = "Account Id" = field("Account Id"), Connector = field(Connector), Scenario = filter(<> Default);
+            }
+        }
+    }
+
+    actions
+    {
+        area(Creation)
+        {
+            action(View)
+            {
+                Image = View;
+                ToolTip = 'View settings for the file account.';
+                ShortcutKey = return;
+                Visible = false;
+
+                trigger OnAction()
+                begin
+                    ShowAccountInformation();
+                end;
+            }
+
+            action(AddAccount)
+            {
+                Image = Add;
+                Caption = 'Add a file account';
+                ToolTip = 'Opens a File Account Wizard setup page in order to add a file account.';
+                Visible = (not IsInLookupMode) and CanUserManageFileSetup;
+
+                trigger OnAction()
+                begin
+                    Page.RunModal(Page::"File Account Wizard");
+
+                    UpdateFileAccounts();
+                end;
+            }
+        }
+        area(Processing)
+        {
+            action(MakeDefault)
+            {
+                Image = Default;
+                Caption = 'Set as default';
+                ToolTip = 'Mark the selected file account as the default account. This account will be used for all scenarios for which an account is not specified.';
+                Visible = (not IsInLookupMode) and CanUserManageFileSetup;
+                Scope = Repeater;
+                Enabled = not IsDefault;
+
+                trigger OnAction()
+                begin
+                    FileAccountImpl.MakeDefault(Rec);
+
+                    UpdateAccounts := true;
+                    CurrPage.Update(false);
+                end;
+            }
+            action(BrowseAccount)
+            {
+                Image = SelectField;
```

Comment:
Do we not have a folder symbol we could use, rather than SelectField?

Reply:
I found and changed it :)